### ISSUE TITLE
Accents in SKU / Name - Not decoded

### ISSUE THREAD

**Issue by jesusvalle** [bhushankumarl/amazon-mws#64]
Greetings. I've noticed that when i request and get the items from a report listing (for example, merchant report), some sku's have accents in it, but not decoded. Example:

"Champú anticaspa" --> "Champ� anticaspa".

In seller central it's displayed properly. This it's happening with product-names too when i get merchant report items.

I've tried decoding the XML in my javascript, but without results.

Thanks so much.

**Comment by marcospampi** [comment]
What's the request you're making against amazon? It is a known issue we encountered in flat files against the charset, not xmls (xml uses UTF8). If  you could try using the `__RAW__` option to get the Buffer object,that you can save with `fs.writeFile` and help us resolve the issue.

**Comment by bhushankumarl** [comment]
@jesusvalle 
https://github.com/bhushankumarl/amazon-mws/blob/master/examples/javaScript/feeds/getFeedSubmissionResultRaw.js

**Comment by jesusvalle** [comment]
I've fixed it passing __RAW__ arg and using iconv to decode the buffer. Thank you.

**Comment by jesusvalle** [comment]
In my file, I'm calling search function like this:
```
.search({
                    Action: 'GetReport',
                    MWSAuthToken: this.amzAccountData.MWSAuthToken,
                    ReportId: requestedReportId,
                    SellerId: this.amzAccountData.sellerId,
                    Version: '2009-01-01',
                    // __RAW__: true
                    __CHARSET__: 'ISO-8859-1'
                })
```
And in amazonMwsResource, i only decode (not encode) the bufferString with the charset sended by param. 
```
 if (userOptions.userCharset && userOptions.userCharset.length > 0) {
                    debug('Inside user Charset option');
                    charset = userOptions.userCharset;
                    try {
                        responseString = iconv.decode(bufferString, charset);
                    } catch (Exception) {
                        debug('Exception iconv ', Exception);
                        return callback.call(self, new Error.AmazonMwsAPIError({
                            message: 'Failed to parse response received from the AmazonMws API',
                            StatusCode: statusCode || 'unknown'
                        }), null);
                    }
                } else {
                    responseString = bufferString.toString();
                }
```
Decoding to ISO-8859-1, now i can see all the accents in my report result.

**Comment by jesusvalle** [comment]
¿You will update the package with those changes in amazazonMwsResource file?

**Comment by bhushankumarl** [comment]
@jesusvalle Yes

**Comment by bhushankumarl** [comment]
Fixed in #66

**Comment by bhushankumarl** [comment]
@jesusvalle Release in latest version https://www.npmjs.com/package/amazon-mws
